packages/desktop-client/src/components/schedules/SchedulesTable.tsx

```diff
@@ -274,7 +311,7 @@ export function SchedulesTable({
         </Row>
       );
     }
-    return renderSchedule({ item });
+    return renderSchedule({ schedule: item as ScheduleEntity });
```

Comment:
Unfortunately, had to cast it as the items as a mix of `ScheduleEntity` & `CompletedScheduleItem`

But not very bad tho, as the implementation still remains the same., 
( except I renamed `type === 'show-completed'    ->       id === 'show-completed `   to satisfy ts 😃 )